src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateCitizenRespondentAosInvitation.java

```diff
@@ -49,17 +44,16 @@
 
             log.info("Generating citizen respondent AoS invitation for case id {} ", caseId);
             final Supplier<Map<String, Object>> templateContentSupplier = templateContent.apply(caseData, caseId, createdDate);
-            final String userAuthorisation = request.getHeader(AUTHORIZATION);
+            final Supplier<String> filename = () -> RESP_AOS_INVITATION_DOCUMENT_NAME + caseId;
```

Comment:
any reason filename is passed as a supplier ?

Reply:
Removed supplier 
https://github.com/hmcts/nfdiv-case-api/pull/582/commits/3d69fcef3e2f6698044af988dc393febb7548193